examples/query/react/basic/src/test/test-utils.tsx

```diff
@@ -0,0 +1,32 @@
+import { render as rtlRender } from '@testing-library/react'
```

Comment:
```suggestion
import { render } from '@testing-library/react'
```
I think this does not need to be renamed any more, right?